rapids_build_backend/impls.py

```diff
@@ -256,11 +298,14 @@ def get_requires_for_build_sdist(config_settings):
             backend := _get_backend(config.build_backend),
             "get_requires_for_build_sdist",
         ):
-            requires.extend(
-                backend.get_requires_for_build_sdist(
-                    _remove_rapidsai_from_config(config_settings)
+            if config.build_backend.startswith("setuptools"):
```

Comment:
Why `startswith` instead of `==`? Are there other setuptools-like backends that have the same name prefix?

Reply:
I was thinking here about two possibilities:

* `setuptools_scm`
* other modules that might be added in the future to `setuptools` (like a `setuptools.build`)

Looks like I was wrong about how `setuptools_scm` is used with PEP 517 builds ... its own docs recommend `build-backend = "setuptools.build_meta"` ([link](https://github.com/pypa/setuptools_scm?tab=readme-ov-file#pyprojecttoml-usage)).

And I'm not sure if that "other modules" even makes sense to care about.

It'd be ok with me to switch this to `== "setuptools.build_meta"`, in the interest of being a bit stricter and not going down this code path if there exist other packages outside of `setuptools` that prefix their names with `setuptools_*`.

I'll change this to an `==`.

Reply:
Just changed all of these to

```python
if config.build_backend == "setuptools.build_meta":
    ...
```